server/cmwell-data-tools/src/main/resources/reference.conf

```diff
@@ -20,6 +20,7 @@ cmwell {
   downloader {
     consumer.http-retry-timeout = "10.seconds"
     consumer.data-fetch-retries = 5
+    consumer.fetch-size = 100
```

Comment:
I think you meant to change the STP length-hint. Changing the config here will change the fetch size for all the data-tools. e.g. bulk-consume will stop to work completely (the server guards again too small chunk, or length-hint is not allowed for bulk-consume - don't remember)

Also the normal consumer should benefit from the 3000 chunk size of consume. Only the STP should be changed. Or am I missing something and everything got broke?

Reply:
Correct @e-orz - my mistake. Not sure I understand what you mean about 'normal consumer'..